File: CS-Examples/22_TextBoxes/ManipulateTextBox.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using System.Data;

using Spire.Xls;
using Spire.Xls.Core;

namespace ManipulateTextBox
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void btnRun_Click(object sender, System.EventArgs e)
        {
            // Create a new workbook object
            Workbook workbook = new Workbook();

            // Load the Excel document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\ManipulateTextBoxControl.xlsx");

            // Get the first worksheet from the workbook
            Worksheet sheet = workbook.Worksheets[0];

            // Get the first textbox from the worksheet
            ITextBox tb = sheet.TextBoxes[0];

            // Change the text of the textbox
            tb.Text = "Spire.XLS for .NET";

            // Set the alignment of the textbox as center
            tb.HAlignment = CommentHAlignType.Center;
            tb.VAlignment = CommentVAlignType.Center;

            // Specify the output file path
            string output = "ManipulateTextBoxControl_out.xlsx";
            // Save the modified workbook to a file
            workbook.SaveToFile(output, ExcelVersion.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            //Launch the Excel file
            ExcelDocViewer(output);
        }
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

    }
}

File: CS-Examples/22_TextBoxes/ManipulateTextBox.vb
Imports System.Collections
Imports System.ComponentModel

Imports Spire.Xls
Imports Spire.Xls.Core

Namespace ManipulateTextBox
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new Workbook instance
            Dim workbook As New Workbook()

            ' Load the workbook from a specified file path
            workbook.LoadFromFile("..\..\..\..\..\..\Data\ManipulateTextBoxControl.xlsx")

            ' Get the first worksheet from the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Get the first TextBox control from the worksheet
            Dim tb As ITextBox = sheet.TextBoxes(0)

            ' Set the text of the TextBox
            tb.Text = "Spire.XLS for .NET"

            ' Set the horizontal alignment of the TextBox to center
            tb.HAlignment = CommentHAlignType.Center

            ' Set the vertical alignment of the TextBox to center
            tb.VAlignment = CommentVAlignType.Center

            ' Specify the output file name
            Dim output As String = "ManipulateTextBoxControl_out.xlsx"

            ' Save the modified workbook to the specified file path in Excel 2013 format
            workbook.SaveToFile(output, ExcelVersion.Version2013)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'Launch the Excel file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
